README.md

```diff
@@ -437,36 +539,42 @@ A client can reduce the operator's `maxLockupPeriod` or `lockupAllowance` after
 
 ### Settlement
 
+Functions for processing payments by moving funds from the client to the payee based on the rail's terms.
+
 #### `settleRail(uint256 railId, uint256 untilEpoch)`
 
-Settles payments for a rail up to a specified epoch.
+This is the primary function for processing payments. It can be called by any rail participant (client, payee, or operator) to settle due payments up to a specified epoch. A network fee in the native token may be required for this transaction.
 
-- **Parameters**:
-  - `railId`: Rail identifier
-  - `untilEpoch`: Target epoch (must not exceed current epoch)
-- **Returns**:
-  - `totalSettledAmount`: Amount transferred
-  - `finalSettledEpoch`: Epoch to which settlement was completed
-  - `note`: Additional settlement information
-- **Requirements**:
-  - Client must have sufficient funds to cover the payment
-  - Client's account must be fully funded _or_ the rail must be terminated
-  - Cannot settle future epochs
+The behavior of `settleRail` critically depends on whether the rail is active or terminated:
+
+-   **For Active Rails:** Settlement can only proceed up to the epoch the client's account was last known to be fully funded (`lockupLastSettledAt`). This is a key safety feature: if a client becomes insolvent, settlement of an active rail halts, preventing it from running a deficit.
+-   **For Terminated Rails:** Settlement can proceed up to the rail's final `endEpoch`, drawing directly from the locked funds that act as a safety guarantee.
```

Comment:
nit: "drawing directly from the streaming lockup"

Reply:
Done.